build.gradle.kts

```diff
@@ -261,6 +261,9 @@ tasks {
         systemProperty("jb.consents.confirmation.enabled", "false")
         systemProperty("ide.mac.file.chooser.native", "false")
         systemProperty("apple.laf.useScreenMenuBar", "false")
+        // VM options to include FUS collector
+        jvmArgs("-Dfus.internal.test.mode=true")
+        jvmArgs("-Didea.is.internal=true")
```

Comment:
Why do we need the FUS properties in `runIdeForUiTests `?

Reply:
discussed during the weekly